blocks/blocks.go

```diff
@@ -166,13 +179,36 @@ func blockDependencies(blk *types.Block) []types.BlockID {
 	return combined
 }
 
+func validateExceptions(block *types.Block, max int) error {
```

Comment:
maybe it's worth returning all the BlockIDs as list for fetching later in the code.

Reply:
why? i think it is simpler and has almost no overhead to use them from the block again